assignments/hackyourtemperature/sources/keys.js

```diff
@@ -0,0 +1,3 @@
+export const keys = {
+  API_KEY: "233947254ea771549eb15ebf5cea869b",
```

Comment:
We usually do not add the key itself to git for security reason (imagine you would pay for any call made with this key, someone could use it and you would be the one paying)

**It is not a blocker for this exercise.**

What you might want to do is either change this line to something like:
```suggestion
  API_KEY: "INSERT_YOUR_KEY_HERE",
```

With the risk that one day someone forget and commits its change into git, or if you have extra time to look into `.env` https://www.npmjs.com/package/dotenv